conf/variant/arp-qtauto/bblayers.conf.sample

```diff
@@ -7,7 +7,6 @@ require conf/variant/common/bblayers-qtauto.conf
 
 BBLAYERS += "\
     ${BSPDIR}/sources/meta-pelux/meta-intel-extras  \
-    ${BSPDIR}/sources/meta-pelux/meta-arp-extras    \
```

Comment:
why?

Reply:
It was only added for the /etc/fstab workaround.